### PR TITLE
Doc updates for 1.4.4 Patch Release

### PR DESCRIPTION
**Pull request by bryantidd**
Doc updates for Release 1.4.4 Patch Release - Changes to master

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.4.4
+Fixes:
+
+- Fixed IngressTrait JWT related issues to allow multiple paths where one path has requestPrincipals and the other doesn't.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: -> `requestPrincipals` (code font); please make the same edit in the next bulleted item.

**Review by rosemarymarano — COMMENTED** [review]
Minor edits but, as far as I know, in the master branch, the 1.4.4 Release Notes should come directly after v1.4.3. Please let me know if you think otherwise.

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (bryantidd) [reply content/en/docs/releasenotes/_index.md:10]: I didn't know if we were going chronologically or numerically

Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:10]: Numerically.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**